collector/lambdacomponents/default.go

```diff
@@ -23,6 +23,7 @@ import (
 	"go.opentelemetry.io/collector/receiver/otlpreceiver"
 	"go.uber.org/multierr"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsprometheusremotewriteexporter"
```

Comment:
The `sigv4auth` extension should be added here rather than the `awsprometheusremotewrite` exporter.

Reply:
I didn't find the right way to add it. Looking at the code in the collector directory and the code in https://github.com/aws-observability/aws-otel-collector regarding the extensions - specifically [this](https://github.com/aws-observability/aws-otel-collector/blob/2d4dd74947dc95826001a82e23a51c47a2d7f43b/pkg/defaultcomponents/defaults.go#L73) , they look different (no ExtensionFactoryMap is present) so I didn't know what the strategy to add extensions in this stripped-down version was. Adding ```awsprometheuswriter``` was straightforward and didn't need major modifications in the existing collector. It looks like we I could to sync the code in https://github.com/aws-observability/aws-otel-collector/tree/2d4dd74947dc95826001a82e23a51c47a2d7f43b/pkg/lambdacomponents with the code here? I think this should be a totally different PR, am I right?

Reply:
You can add the ExtensionFactoryMap in this PR, and add the `sigv4` extension instead of the `awsprometheusremotewrite` exporter, it doesn't have to be in a separate PR